### PR TITLE
Conditionally return the correct funding source, based on whether a p…(2.2.0)

### PR DESCRIPTION
**Pull request by jleonardw9**
…rotocol is pending funding or not.

### CODE REVIEW

File: app/models/protocol.rb [hunk 1]
```diff
@@ -96,6 +97,14 @@ def protocol_type
     sparc_protocol.type
   end
 
+  def sparc_funding_source
```
Comment (amcates) [inline app/models/protocol.rb]: def sparc_funding_source
  funding_source.blank? ? potential_funding_source : funding_source
end

Comment (amcates) [inline app/models/protocol.rb]: Just a suggestion

Comment (jleonardw9) [inline app/models/protocol.rb]: That's better, thanks.

**Comment by wtholt** [comment]
Looks good to me

**Pull request merged**